### DISCUSSION TITLE
Order.paid webhook

### DISCUSSION THREAD

**Discussion by FutureBuddha** [swellstores/community#554] [Help] [answered]
Hi there,

I am working on a project where I need to send via webhook the amount a user has paid for a product. My goal is to collect and save the total amount each user pays for specific products.

I attempted to use the `order.paid` webhook, but I only receive the `payment_id` and `id` in the response.

Is there a way to get the amount, user ID, and product ID directly when the order is paid? This information is crucial for tracking payments accurately in another application.

Thank you very much in advance for your assistance.

Best regards and thank you very much in advance

**Comment by logeshswell** [comment, marked as answer]
`order.paid` webhook is the right one for your usecase, but you need to fetch the payment and order details to track those values.

**Reply by logeshswell** [reply]
https://developers.swell.is/backend-api/payments/retrieve-a-payment
https://developers.swell.is/backend-api/orders/retrieve-an-order

**Reply by FutureBuddha** [reply]
I thought so. I was hoping that I could handle this with one request using the webhook, because then it would be from the Swell webhook to my app, and from the app back to Swell to get the data.

Is it possible to just add the data to the webhook?

**Reply by logeshswell** [reply]
No, sorry it's not possible to add custom data to a standard webhook.
